service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -0,0 +1,41 @@
+[id="updating-ossm"]
+= Updating {ProductName} from version 1.0.1 to 1.0.2
+include::modules/ossm-document-attributes.adoc[]
+:context: installing-ossm
+toc::[]
+
+Updating {ProductName} requires extra steps before you update OpenShift to version 4.2. 
+You must upgrade {ProductName} to 1.0.2 before upgrading OpenShift 4.1.x to 4.2.
+
+.Prerequisites
+
+* {ProductName} version 1.0.1
+* OpenShift version 4.1
+
+.Procedure
+
+. Configure existing SMCP resource requests by running the following patch command. Replace
+the <smcp namespace> and <smcp name> with your specific names.
++
+----
+$ oc patch -n <smcp namespace> smcp <smcp name e.g.basic-install> --type=merge -p '{"spec": {"istio": {"global": {"defaultResources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}},"proxy": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}}},"defaultPodDisruptionBudget": {"enabled": false}},"security": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"galley": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"pilot": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"mixer": {"telemetry": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"gateways": {"istio-egressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"istio-ingressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"prometheus": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}}}}'
```

Comment:
Consider chunking it up with line breaks[1] so it's more easily readable (even if the spec blob stays on its own line)? This will also allow for more targeted callouts (for example, the "e.g. basic-install" could then be a callout). E.g.:

```
----
$ oc patch -n <smcp_namespace> smcp <smcp_name> \// <1>
    --type=merge -p \
    '{"spec": {"istio": {"global": {"defaultResources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}},"proxy": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}}},"defaultPodDisruptionBudget": {"enabled": false}},"security": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"galley": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"pilot": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"mixer": {"telemetry": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"gateways": {"istio-egressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"istio-ingressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"prometheus": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}}}}'
----
<1> For example, `basic-install`.
```

Re: `\// <1>`, usually a simple `<1>` would be fine, but since we don't want whitespace after the `\` line break, this makes it more copy-paste friendly. [2]

[1] https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks
[2] https://asciidoctor-docs.netlify.com/asciidoc/1.5/source/callout/